cadctl/cmd/investigate/investigate.go

```diff
@@ -111,7 +110,7 @@ func run(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	ocmClient, err := GetOCMClient()
+	ocmClient, err := ocm.New("")
```

Comment:
Wouldn't it make sense to remove the string argument to `New` given that we aren't using it anymore?

Reply:
It would, but I would rather do it in followup, when I clean up the client section. I prefer to keep this as is to make the changes limited.